module/Application/src/Application/Service/AclFactory.php

```diff
@@ -47,6 +49,7 @@ public function createService(ServiceLocatorInterface $serviceLocator)
 		$acl->allow(User::ROLE_USER, 'Ora\Task', 'TaskManagement.Task.accept', new TaskOwnerAndCompletedTaskWithEstimationProcessCompletedAssertion());
 		$acl->allow(User::ROLE_USER, 'Ora\Task', 'TaskManagement.Task.assignShares', new TaskMemberAndAcceptedTaskAssertion());
 
+		$acl->allow(null, null, 'Application.Host.allowLocalhost', new HttpHostLocalhostAssertion());
```

Comment:
"[...] l'operazione cui mi riferisco è quella di controllare se l'host è localhost [...]" si chiama criterio di accesso ed è espresso da una Assertion, che si chiama infatti `HttpHostLocalhostAssertion`.

In una ACL si mappano criteri di accesso su operazioni su risorse come puoi anche vedere dalle righe precedenti nel file in questione. Il terzo parametro deve identificare l'operazione.